test/terra/backends/test_qasm_mps_simulator.py

```diff
@@ -0,0 +1,86 @@
+from test.terra.backends.qasm_simulator.qasm_measure import QasmMultiQubitMeasureTests
+from test.terra.backends.qasm_simulator.qasm_cliffords import QasmCliffordTests
+from test.terra.backends.qasm_simulator.qasm_cliffords import QasmCliffordTestsWaltzBasis
+from test.terra.backends.qasm_simulator.qasm_cliffords import QasmCliffordTestsMinimalBasis
+from test.terra.backends.qasm_simulator.qasm_noncliffords import QasmNonCliffordTests
+from test.terra.backends.qasm_simulator.qasm_noncliffords import QasmNonCliffordTestsWaltzBasis
+from test.terra.backends.qasm_simulator.qasm_noncliffords import QasmNonCliffordTestsMinimalBasis
+from test.terra.backends.qasm_simulator.qasm_unitary_gate import QasmUnitaryGateTests
+from test.terra.backends.qasm_simulator.qasm_initialize import QasmInitializeTests
+# Conditional instruction tests
+from test.terra.backends.qasm_simulator.qasm_conditional import QasmConditionalGateTests
+from test.terra.backends.qasm_simulator.qasm_conditional import QasmConditionalUnitaryTests
+from test.terra.backends.qasm_simulator.qasm_conditional import QasmConditionalKrausTests
+# Algorithm circuit tests
+from test.terra.backends.qasm_simulator.qasm_algorithms import QasmAlgorithmTests
+from test.terra.backends.qasm_simulator.qasm_algorithms import QasmAlgorithmTestsWaltzBasis
+from test.terra.backends.qasm_simulator.qasm_algorithms import QasmAlgorithmTestsMinimalBasis
+# Noise model simulation tests
+from test.terra.backends.qasm_simulator.qasm_noise import QasmReadoutNoiseTests
+from test.terra.backends.qasm_simulator.qasm_noise import QasmPauliNoiseTests
+from test.terra.backends.qasm_simulator.qasm_noise import QasmResetNoiseTests
+from test.terra.backends.qasm_simulator.qasm_noise import QasmKrausNoiseTests
+# Snapshot tests
+from test.terra.backends.qasm_simulator.qasm_snapshot import QasmSnapshotStatevectorTests
+from test.terra.backends.qasm_simulator.qasm_snapshot import QasmSnapshotStabilizerTests
+# Other tests
+from test.terra.backends.qasm_simulator.qasm_method import QasmMethodTests
+
+
+class TestQasmMatrixProductStateSimulator(
+        common.QiskitAerTestCase,
+        #QasmMethodTests,  # CRASHES: segmentation fault
+        QasmMeasureTests,  
+        QasmMultiQubitMeasureTests,
+        QasmResetTests,
+        QasmConditionalGateTests,
+        QasmConditionalUnitaryTests,
+        QasmCliffordTests,
+        QasmCliffordTestsWaltzBasis,
+        QasmCliffordTestsMinimalBasis,
+        QasmNonCliffordTests,
+        QasmNonCliffordTestsWaltzBasis,
+        QasmNonCliffordTestsMinimalBasis,
+        QasmAlgorithmTests,
+        QasmAlgorithmTestsWaltzBasis,
+        QasmAlgorithmTestsMinimalBasis,
+        QasmUnitaryGateTests,
+        #QasmInitializeTests,  # CRASHES: uncaught exception
+        QasmReadoutNoiseTests,
+        QasmPauliNoiseTests,
+        QasmResetNoiseTests,
+        QasmSnapshotStatevectorTests,
+        QasmSnapshotStabilizerTests,
```

Comment:
I forgot to add the `QasmSnapshotProbabilitiesTests` which was recently merged to master